packages/graph-explorer/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+coverage/
```

Comment:
node_modules - is it included in the top-level gitignore?

Suggesting adding it to the main ignore to keep gitignore in one place.

Reply:
Unfortunately the repo is set up strangely. The gitignore is at the root, but the prettier and eslint configs are inside `packages/graph-explorer`.

VS Code prettier integration will only work if I open the editor to the where the prettier config lives. But then the gitignore is not read and I end up with a lot of noise in my VS code file searches since it includes node modules and coverage results.

Ultimately, the solution is to reorganize the folder structure to be more friendly to developers working on the repository. But for the short term I wanted a solution that doesn't rock the boat too much but solves the problem.

If it is too much of a disruption I can remove it, but it is not harming anything to my knowledge.